samples/todo/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
+
+This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
```

Comment:
You might want to explain what this site/component is, as at first I thought this was a directory of things that need done.